packages/dev/core/src/Collisions/gpuPicker.ts

```diff
@@ -400,12 +460,116 @@ export class GPUPicker {
         });
     }
 
-    private async _readTexturePixelsAsync(x: number, y: number) {
+    // pick multiple pixels
+    private _executeMultiPicking(
+        xy: { x: number; y: number }[],

```

Comment:
Could the type be IVector2Like[] ? for extra compat

Reply:
Changed. Three years with bjs and never came across this interface. Thanks!

Reply:
Not seeing it changed here ?